dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -0,0 +1,372 @@
+﻿#include "pch.h"
+#include "AppNotificationBuilder.h"
+#include <winrt/Windows.Globalization.h>
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationBuilder.g.cpp"
+#include <fmt/core.h>
```

Comment:
What's fmt/core.h ?

Reply:
Removing as it was a header for the fmt library